src/main/kotlin/com/pubnub/api/PNConfiguration.kt

```diff
@@ -404,11 +403,11 @@ open class PNConfiguration(
      * Defaults to [RequestRetryPolicy.None].
      */
 
-//    var newRetryPolicy: RequestRetryPolicy = RequestRetryPolicy.None
-    var newRetryPolicy: RequestRetryPolicy = RequestRetryPolicy.Linear(
-        delayInSec = 3,
-        maxRetryNumber = 3,
-        excludedOperations = listOf(RetryableEndpointGroup.SUBSCRIBE)
-    )
+    var newRetryPolicy: RequestRetryPolicy = RequestRetryPolicy.None
+//    var newRetryPolicy: RequestRetryPolicy = RequestRetryPolicy.Linear(
```

Comment:
clean this up before merging?

Reply:
Yes. I though initially that I will add to this PR code related to new parameter but now I think for this PR it can be `**internal** var newRetryPolicy: RequestRetryPolicy = RequestRetryPolicy.None`
and then in separate PR I will rename newRetryPolicy and make it public.